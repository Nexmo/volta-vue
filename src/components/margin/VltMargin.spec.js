import { shallowMount } from '@vue/test-utils';
import { VltMargin } from '..';

describe('vlt-margin', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(VltMargin);
  });

  test('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  test('renders with a top margin of 2', async () => {
    wrapper.setProps({ top: 2 });
    await wrapper.vm.$nextTick();
    expect(wrapper.element).toMatchSnapshot();
  });

  test('renders with a top margin of 2 on small screens', async () => {
    wrapper.setProps({ sTop: 2 });
    await wrapper.vm.$nextTick();
    expect(wrapper.element).toMatchSnapshot();
  });
});
