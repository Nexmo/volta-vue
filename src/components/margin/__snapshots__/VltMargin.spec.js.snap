// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vlt-margin renders correctly 1`] = `
<div
  class=""
/>
`;

exports[`vlt-margin renders with a top margin of 2 1`] = `
<div
  class="Vlt-margin--top2"
/>
`;

exports[`vlt-margin renders with a top margin of 2 on small screens 1`] = `
<div
  class="Vlt-margin--S-top2"
/>
`;
